app/models/game.rb

```diff
@@ -36,6 +36,17 @@ def check?
     kings_in_check?('white') || kings_in_check?('black')
   end
 
+  def forfeit(current_user)
+    if current_user.id == white_player_id
+      white_player_id.update_attributes(white_player_id.losses: white_player_id.losses + 1)
+      black_player_id.update_attributes(black_player_id.wins: black_player_id.wins + 1)
+    else
+      white_player_id.update_attributes(white_player_id.wins: white_player_id.wins + 1)
+      black_player_id.update_attributes(black_player_id.losses: black_player_id.losses + 1)
+    end
+    white_player_id, black_player_id = nil, nil # end the game and make unavailable
```

Comment:
I don't think we want to remove the players. Because we want to have a record of all the game they have played.

Do we have a status for `game`? We could change the game status to be over.

We could have a `winning_player_id` on `Game`. And we set that to the winner.